### PR TITLE
Fix Login component - infinite spinning on close Blockstack modal

### LINKED ISSUE #611: Closing Get Started Dialog keeps you stuck at loading
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in QA
    * Followed all applicable steps in the TROUBLESHOOTING guide: TODO
    * Checked for relevant discussions on Discord: <https://discord.gg/pPERUuv>
    * Checked that your issue isn't already filed: <https://github.com/COVID-19-electronic-health-system/Corona-tracker/issues>

### Description

When initially coming to the app and clicking get started a popup shows up. If you close that pop up the app is stuck on the Loding component with no way of getting out except for refreshing.

loding

### Steps to Reproduce

1.  Start at Login page (sign out)
2.  Click Get Started
3.  Click the x on the top right

**Expected behavior:**
Probably should go back to the Login component

**Actual behavior:**

Is stuck on Loding component
**Reproduces how often:**
 100%

### Versions

1.0.10

### Additional Information

could remove the loding page since its only used there 
or
we could move setLoading(true) somewhere else instead of onclick of the 'Get Started' button since there is no way of setting loading to false after.

### PR DESCRIPTION
**Pull request by pavel-ilin**
fixes #611

### Please provide enough information so that others can review your pull request:

Add event listener `onPointerOver` for parent `div` element in `Login` component. On event function `closeModal` invokes and reset value of the `loginLoading` variable. Which prevent infinite loading spin when we close Blockstack modal without login in.

### Final Checklist

- [x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [x] Have you added any new tests necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x] Did you use yarn, not npm?
- [x] Did you use Material-UI wherever possible?
- [x] Did you format according to Prettier?
- [x] Did you run all of your most recent changes locally to make sure everything is working?

### CODE REVIEW

**Comment by acthelemann** [comment]
I checked out your branch and reproduced it:

spinz

**Comment by pavel-ilin** [comment]
You are right about mobile version. Can't think about anything at the moment. It's tricky because we not control the Blockstack modal.

Will do more research.

**Comment by acthelemann** [comment]
Hmm I'd hope there's something in the Blockstack API to tell when this happens.

If not, I'm not so sure that this loading screen is even necessary, but I'd like some others' thoughts before removing it. @AdhamAH @SomeMoosery

**Comment by SomeMoosery** [comment]
I've been thinking about this, and was thinking maybe we only display it on desktop (or we could remove it entirely, frankly).

On mobile, it adds nothing, and on desktop the loading icon is hidden behind the Blockstack modal which serves to just hide the "Get Started" screen.

I'd honestly be fine removing it altogether, it seems to be more headache than it's worth

**Comment by pavel-ilin** [comment]
I think we can remove it. It's not add aesthetic or functional value. If everyone's agree, I can do it.

**Comment by SomeMoosery** [comment]
Yeah, let's send it 🏂 we can just keep removing it in this PR, and it'll close out the issue 👍

File: client/src/components/Login.jsx [hunk 1]
```diff
@@ -55,9 +55,12 @@ const Login = props => {
     setLoading(true);
     doOpenAuth();
   };
+  const closeModal = () => {
+    setLoading(false);
+  };
```
Comment (SomeMoosery) [inline client/src/components/Login.jsx:60]: I think `setLoading` only deals with the loading component itself, right? We could probably just remove this functionality entirely (from Redux as well) if we're going to remove the loading screen.

Comment (pavel-ilin) [reply client/src/components/Login.jsx]: All references `setLoading`  removed from the app.

**Comment by pavel-ilin** [comment]
> looks good but you need to update the package.json, sorry I just merged a PR

Version updated.

**Pull request merged**